src/vivarium_cluster_tools/psimulate/utilities.py

```diff
@@ -71,13 +69,12 @@ def get_output_directory(model_specification_file=None, output_directory=None, r
         output_directory = root / model_specification_name / launch_time
     return output_directory
 
-DIRECTORY_PERMS=0o775
```

Comment:
Instead of pulling this up to globals, make it a function local variable and adjust the docstring to say what it sets.